src/Core/Components/MessageBar/FluentMessageBar.razor

```diff
@@ -107,16 +107,16 @@
                 {
                     <FluentAnchor Href="@(String.IsNullOrEmpty(Link?.Href) ? "#" : Link?.Href)"
                                   Appearance="Appearance.Hypertext"
-                                  Target="_blank"
-                                  @onclick="@LinkClickedAsync"
+                                  Target="@(Link?.Target ?? "_blank")"
+                                  OnClick="@LinkClickedAsync"
                                   title="Link">
                         @Link?.Text
                     </FluentAnchor>
                 }
                 @if (ShowPrimaryAction)
                 {
-                    <FluentButton Appearance="Appearance.Hypertext"
-                                  @onclick="@PrimaryActionClickedAsync"
+                    <FluentButton Appearance="Appearance.Neutral"
```

Comment:
Why did you changed the `Appearance`?

Reply:
The FluentButton throws an ArgumentException when Appearance.Hypertext is used (FluentButton.razor.cs line 174, "Appearance.Hypertext is not supported for FluentButton").  I did see that other FluentButtons (for exmaple FluentMessageBar line 44) also use Neutral.

Reply:
OK. Makes sense. The design won't be the same in the notification center, but why not.